com.unity.render-pipelines.universal/Shaders/Particles/ParticlesLit.shader

```diff
@@ -20,11 +20,6 @@ Shader "Universal Render Pipeline/Particles/Lit"
         _EmissionMap("Emission", 2D) = "white" {}
         _ReceiveShadows("Receive Shadows", Float) = 1.0
 
-        _ClearCoat("Clear Coat", Float) = 0.0
```

Comment:
So i guess there would need to be a ComplexLitParticles or similar, to get this feature back in for particles in the future?

Reply:
No. I think the plan is to take a step back on that, possibly support it via shader graph in the future.